lib/rust/profiler/data/src/bin/devtools.rs

```diff
@@ -0,0 +1,131 @@
+//! Tool that generates Chrome DevTools-compatible files from profiling interval data.
+//!
+//! # Usage
+//!
+//! The tool reads a JSON-formatted event log from stdin, and writes a report to stdout.
+//!
+//! For example:
+//!
+//! ```console
+//! ~/git/enso/data $ cargo run --bin intervals < profile.json > devtools.json
+//! ```
```

Comment:
These docs don't explain several important things:
1. Where is the spec this code bases on? Where are the descriptions of the format supported by Chrome, so we can easily refer to it in the future when extending this code? (ofc this can be provided in other docs than module docs, but I haunt found it below).
2. What is `profile.json`. I know the context so I understand it, but someone reading this code might not. Such docs should explain the context, including short info of what profile.json is, how to generate it, and should provide step-by step guide of how to get it running starting with 0 knowledge.

Reply:
1. Official documentation is unavailable. I've added a link to a Google Doc I found that seems to be someone's reverse-engineering of the format.
2. I've added a link to the appropriate section of the design doc. The design doc is out of date wrt. the procedure for capturing the data since we decided to implement an interactive workflow before we implement batch mode, so I'll write a PR for the design doc repo now to describe the capture procedure as implemented.